docs/ErDiagram.puml

```diff
@@ -0,0 +1,30 @@
+@startuml
+
+title Entity Relationship Diagram 
+' hide the spot
+hide circle
+
+' avoid problems with angled crows feet
+skinparam linetype ortho
+
+entity "Consent" as e01 {
```

Comment:
I think this can just be `entity Consent {` and then `Consent ||..o{ Scope` down below. Might be easier to read.